assets/price-per-item.js

```diff
@@ -73,7 +73,7 @@ if (!customElements.get('price-per-item')) {
 
         // updatedCartQuantity is undefined when qty is updated on product page. We need to sum entered qty and current qty in cart.
         // updatedCartQuantity is not undefined when qty is updated in cart. We need to sum qty in cart and min qty for product.
-        this.currentQtyForVolumePricing = updatedCartQuantity === undefined ? this.getCartQuantity(updatedCartQuantity) + enteredQty : this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.min);
+        this.currentQtyForVolumePricing = updatedCartQuantity === undefined ? this.getCartQuantity(updatedCartQuantity) + enteredQty : this.getCartQuantity(updatedCartQuantity) + parseInt(this.input.step);
```

Comment:
Using `step` instead of `min` works for every cases except one. Let's say you have some qty in the cart and then you remove it. In this scenario  `this.currentQtyForVolumePricing` will be equal to 1 (step) but should be equal to 5 (min). On the other hand, we still see the correct price because we cannot have a lower price than the price for min rule.

Reply:
From my understanding vol pricing rules must respect qty rules (vol pricing min qty needs to be equal to or higher than qty rules)

We should make sure to test this in variant list where you can also remove items.